src/observe-tag.js

```diff
@@ -13,10 +12,23 @@ const byDistanceFromIndex = targetIndex => (indexA, indexB) => {
 	return diffFromTargetA - diffFromTargetB
 }
 
+const hasMatchingTagName = tagName => node => {
+	if (node.tagName === tagName) {
+		return NodeFilter.FILTER_ACCEPT
+	}
```

Comment:
You have a one liner here, but you wrap it in curly braces, but lower in the code, you do the one line return sans curly. I would keep it consistent